appcues/src/main/java/com/appcues/DeepLinkHandler.kt

```diff
@@ -142,12 +151,14 @@ internal class DeepLinkHandler(scope: AppcuesScope) {
         if (!extras.getBoolean(NOTIFICATION_TEST_EXTRA)) {
             analyticsTracker.track(
                 PushOpened.eventName,
-                properties = mapOf(
+                properties = mapOf<String, Any?>(
                     "notification_id" to extras.getString(NOTIFICATION_ID_EXTRA, null),
-                    "workflow_id" to extras.getString(NOTIFICATION_WORKFLOW_ID, null),
-                    "workflow_task_id" to extras.getString(NOTIFICATION_WORKFLOW_TASK_ID, null),
+                    "push_notification_version" to extras.getInt(NOTIFICATION_VERSION_EXTRA, -1).let { if (it == -1) null else it },
```

Comment:
and then this needs to change to `getLong` and compare to `-1L` (and L159 below)